src/services/YotiCallbackProcessor.ts

```diff
@@ -127,10 +127,10 @@ export class YotiCallbackProcessor {
   		const { credentialSubject, evidence } =
         this.generateVerifiableCredential.getVerifiedCredentialInformation(yotiSessionID, completedYotiSessionInfo, documentFields);
 
-			if (!credentialSubject || !evidence) {
-				this.logger.error({ message: "Missing Credential Subject or Evidence payload" }, { credentialSubject, evidence });
-				throw new AppError(HttpCodesEnum.SERVER_ERROR, "Missing Credential Subject or Evidence payload");
-			}
+  		if (!credentialSubject || !evidence) {
+  			this.logger.error({ message: "Missing Credential Subject or Evidence payload" }, { credentialSubject, evidence });
+  			throw new AppError(HttpCodesEnum.SERVER_ERROR, "Missing Credential Subject or Evidence payload");
```

Comment:
can we introduce a `messageCode` for this?

Reply:
Accidentally formatted an incorrect file, changes have been reverted